faqs.html

```diff
@@ -148,6 +150,73 @@ <h2 class="process__title title">Frequently Asked Questions</h2>
             }
           });
         });
+      });
+            // to change the color of the icon when clicked

```

Comment:
```suggestion
        

```

Reply:
Should I delete the part and merge again?